src/site/stages/build/plugins/add-asset-hashes.js

```diff
@@ -1,69 +1,85 @@
-function addAssetHashes(isPreview = false) {
-  // In non-development modes, we add hashes to the names of asset files in order to support
-  // cache busting. That is done via WebPack, but WebPack doesn't know anything about our HTML
-  // files, so we have to replace the references to those files in HTML and CSS files after the
-  // rest of the build has completed. This is done by reading in a manifest file created by
-  // WebPack that maps the original file names to their hashed versions. Metalsmith actions
-  // are passed a list of files that are included in the build. Those files are not yet written
-  // to disk, but the contents are held in memory.
+/* eslint-disable no-continue, no-param-reassign */
 
-  return (files, metalsmith, done) => {
-    // Read in the data from the manifest file.
-    const manifestKey = Object.keys(files).find(
-      filename => filename.match(/file-manifest.json$/) !== null,
-    );
+const environments = require('../../../constants/environments');
 
-    const originalManifest = JSON.parse(files[manifestKey].contents.toString());
+const FILE_MANIFEST_FILENAME = 'generated/file-manifest.json';
+
+function copyAssetsToTeamSitePaths(buildOptions, files, entryNamesDictionary) {
+  // TeamSite pages such as "benefits.va.gov" have hardcoded references to certain
+  // JavaScript bundles so that we can inject our header/footer into pages outside
+  // of our source. Therefore, we can't apply a hash to those bundles.
+  const teamsiteInjectionBundles = {
```

Comment:
What do you think of renaming this to `teamSiteInjectionBundles` (capital `S`) or renaming `teamSitePath` with a lowercase `s` (`teamsitePath`) for consistency?

Reply:
Fine with me.